### PR TITLE
Feature/dapp 319

### PR DESCRIPTION
**Pull request by leodhb**
# Description

- The integrations list for the manager
image

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] It is urgent
- [ ] <span style="color:#d44037">Dangerous change</span>

### CODE REVIEW

File: src/hooks/useIntegrations/index.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import integrationsApi from "services/api/integrationsApi";
+
+function useIntegrations() {
```
Comment (nathielymm) [inline src/hooks/useIntegrations/index.ts:3]: Maybe this hook could be inside hooks/apiHooks, what do you think?

Comment (leodhb) [reply src/hooks/useIntegrations/index.ts:3]: Sure. We have some apiHooks outside this context, such as useUsers. Maybe we need to consider a refactor for these ones

Comment (leodhb) [reply src/hooks/useIntegrations/index.ts:3]: I gonna move the useIntegrations :)

Comment (nathielymm) [reply src/hooks/useIntegrations/index.ts]: Yeah! useUsers needs a refactor

**Pull request merged**